commands/station.js

```diff
@@ -24,6 +25,8 @@ export const station = async () => {
     process.exit(1)
   }
 
+  startPingLoop().unref()
```

Comment:
there were some race conditions where the telemetry `ping` loop started when it shouldn't, causing crashes due to the `assert(process.env.FIL_WALLET_ADDRESS)`. This makes it only start when we want it